WordPress/WordPressUITests/Utils/XCTest+Extensions.swift

```diff
@@ -16,12 +16,11 @@ extension XCUIElement {
      */
     func clearTextIfNeeded() -> Void {
         let app = XCUIApplication()
-        let content = self.value as! String
 
-        if content.count > 0 && content != self.placeholderValue {
-            self.press(forDuration: 1.2)
-            app.menuItems.element(boundBy: 1).tap() // Select All (in the text edit menu)
-            app.menuItems.element(boundBy: 0).tap() // Cut (in the text edit menu)
+        self.tap()
+
+        for _ in 0...(self.stringValue?.count ?? 0) {
+            app.keys["delete"].tap()
```

Comment:
I noticed that deleting each character individually is fairly slow — what do you think of something like this instead:

```
if (self.stringValue?.count ?? 0) > 0 {
    self.press(forDuration: 1.2)
    app.keys["delete"].tap()
}
```

Reply:
Note that my suggestion is a full replacement of the changes here, so no `self.tap()` first — the long press without tapping on the text field will select all the text.

Reply:
This works much better! :) I'll update the PR!